src/openrct2/interface/Viewport.cpp

```diff
@@ -1277,17 +1277,16 @@ void viewport_set_visibility(uint8_t mode)
  */
 static bool PSSpriteTypeIsInFilter(paint_struct* ps, uint16_t filter)
 {
-    if (ps->sprite_type == VIEWPORT_INTERACTION_ITEM_NONE
-        || ps->sprite_type == 11 // 11 as a type seems to not exist, maybe part of the typo mentioned later on.
```

Comment:
This code was a bit weird before (the `== 11`), it looks more correct now, but I'm commenting to see what other people think

Reply:
This is checking for the types that can't be interacted with. So thats: None, labels and anything invalid (> banner)